changelog.txt

```diff
@@ -2186,7 +2186,7 @@ Note: OpenTTD was migrated to GitHub for 1.9, so SVN revision and FlySpray numbe
 - Fix: [NewGRF] Additional text in fund industry window is NewGRF supplied and thus should have a default colour (r22631)
 - Fix: Also initialise _old_vds with newgame settings; TTD savegames do not contain these settings [FS#4622] (r22626)
 - Fix: Do not zero the orders of disaster vehicles when converting savegames [FS#4642] (r22625)
-- Fix: When closing an AI company the local player cheated to, we need to cheat him to another company [FS#4654] (r22624, r22623)
+- Fix: When closing an AI company the local player cheated too, we need to cheat them to another company [FS#4654] (r22624, r22623)
```

Comment:
"too" here is incorrect in this context. The original spelling of "to" is correct.